DUnitX.TestFramework.pas

```diff
@@ -249,12 +259,20 @@   TTestFixtureInfoList = class(TDUnitXList<ITestFixtureInfo>, ITestFixtureInfoLi
     function GetExceptionLocationInfo : string;
     function GetExceptionAddressInfo : string;
     function GetExceptionAddress : Pointer;
+    function GetIsComparable: boolean;
+    function GetExpected: string;
+    function GetActual: string;
+    function GetFormat: TDUnitXComparableFormatClass;
 
     property ExceptionClass : ExceptClass read GetExceptionClass;
     property ExceptionMessage : string read GetExceptionMessage;
     property ExceptionLocationInfo : string read GetExceptionLocationInfo;
     property ExceptionAddressInfo : string read GetExceptionAddressInfo;
     property ExceptionAddress : Pointer read GetExceptionAddress;
+    property IsComparable: boolean read GetIsComparable;
+    property Expected: string read GetExpected;
+    property Actual: string read GetActual;
+    property Format: TDUnitXComparableFormatClass read GetFormat;
```

Comment:
Imo these properties should not be directly exposed because they are only valid for a special type of error. Rather expose the exception object which then exposes its special properties (like for being `ETestFailureStrCompare`).